### ISSUE TITLE
Files uploaded from 2021 on are zipped using a new proprietary compression algorithm

### ISSUE THREAD

**Issue by HFAwesomeCharts** [Nonprofit-Open-Data-Collective/irs-990-data-issue-tracker#2]
Files uploaded from 2021 on are zipped using a new proprietary compression algorithm. The IRS is now using a compression algorithm called Deflate64 that isn't supported by Python out of the box. (It's the default on the new Windows OS so the IRS may not even be aware that it's proprietary.) If you're using Python, you have to install Deflate64 before you can automatically download and unzip the files.

**Comment by lecy** [comment]
Any tips on which Deflate64 program to use? 

https://pypi.org/project/zipfile-deflate64/  ??

**Comment by HFAwesomeCharts** [comment]
Yes, that is the one I installed. I used the command "pip install zipfile-deflate64" to install it. I am on a Windows machine, too, though...

**Comment by HFAwesomeCharts** [comment]
Tagging @tone711 to make sure I'm answering this correctly...

**Comment by tone711** [comment]
@HFAwesomeCharts this looks great. If Python and .Net (System.IO.Compression) don't support Deflate64 out of the box, it shows the more significant extent of the issue.
